From fe304b186408d9228e2126427aa492029b18d57b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mathieu=20Dupr=C3=A9?= <mathieu.dupre@savoirfairelinux.com>
Date: Fri, 3 Oct 2025 17:24:34 +0200
Subject: [PATCH] Revert "Properly close connection to Rados"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit 5755b088a2a56a4ef512fc64ac7103bbdb88ee13.

We need to revert this patch because it is imcompatible with our patch.

Sould be removed in the 2.0.0 release.

Upstream-Status: Inappropriate [oe specific]
Signed-off-by: Mathieu Dupré <mathieu.dupre@savoirfairelinux.com>
---
 vm_manager/helpers/rbd_manager.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/vm_manager/helpers/rbd_manager.py b/vm_manager/helpers/rbd_manager.py
index e427275..de259d4 100644
--- a/vm_manager/helpers/rbd_manager.py
+++ b/vm_manager/helpers/rbd_manager.py
@@ -69,7 +69,6 @@ class RbdManager:
         Close I/O context.
         """
         self._ioctx.close()
-        self._cluster.shutdown()

     # Namespace methods
     def list_namespaces(self):
--
2.51.0
